utils/merge.py

```
@@ -160,37 +237,31 @@ def make_author_model(book, authors: list[dict[str, str]]):
-                f"No author ASIN for: "
-                f"{author_name_full}"
-            )
+            logger.warning(f"No author ASIN for: {author_name_full}")
 
         # Check if author is in database
         if not (author := Author.objects.filter(**_filter_vals).first()):
-            logger.info(
-                f"Using existing db entry for author: "
-                f"{author_name_full}"
-            )
+            logger.info(f"Using existing db entry for author: {author_name_full}")
```

Ruff (0.15.0)

[warning] Logging statement uses f-string

(G004)